### PR TITLE
chore: update paragon and frontend-build in profile

### PR DESCRIPTION
**Pull request by Yagnesh1998**
There is an edx organization on NPM that currently publishes many packages. An organization can't be renamed but there is a process for moving packages between organizations. The edx organization or scope as NPM calls it, has both packages that are a part of the Open edX ecosystem as well as packages that are specific to edx.org. To reduce confusion we'll be moving the openedx packages to a new openedx scope in NPM.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Yagnesh1998! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by arbrandes** [comment]
@Yagnesh1998, mind resolving conflicts?  Thanks!

**Comment by jsnwesson** [comment]
Just noting this for Aperture's review since this _might_ require a follow-up change in our stage/prod build config once this is merged.

**Review by MaxFrank13 — APPROVED** [review]

**Review by arbrandes — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -29,16 +29,16 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-footer": "13.0.3",
+    "@edx/frontend-component-footer": "13.0.2",
```
Comment (arbrandes) [inline package.json:32]: Why are we downgrading the footer?

File: package.json [hunk 2]
```diff
@@ -29,16 +29,16 @@
...
     "@edx/frontend-component-header": "5.0.2",
-    "@edx/frontend-platform": "7.1.1",
+    "@edx/frontend-platform": "7.0.1",
```
Comment (arbrandes) [inline package.json:34]: Why downgrade frontend-platform?

**Review by arbrandes — CHANGES_REQUESTED** [review]
Trying to understand the decisions, here.

File: package.json (hunk 1 above)
Comment (Yagnesh1998) [reply package.json]: there may be some misunderstanding

**Comment by justinhynes** [comment]
@Yagnesh1998 Just another bump requesting conflict resolution. Would you be able to address any of the questions/comments on the PR too? TIA!

**Comment by deborahgu** [comment]
@Yagnesh1998 is this PR still in process? Will you be able to resolve the conflicts?

@arbrandes  I am pretty sure that @Yagnesh1998  resolved your questions (by fixing the accidentally downgraded packages); am I correct?

**Comment by openedx-webhooks** [comment]
@Yagnesh1998 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-profile&entry.867055334=https://github.com/openedx/frontend-app-profile/pull/975&entry.1484655318=https://github.com/Yagnesh1998&entry.752974735=2024-02-27+09:27&entry.1917517419=2024-04-22+20:07&entry.2133058324=No) so we can improve your experience in the future.

**Comment by openedx-webhooks** [comment]
@Yagnesh1998 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-profile&entry.867055334=https://github.com/openedx/frontend-app-profile/pull/975&entry.1484655318=https://github.com/Yagnesh1998&entry.752974735=2024-02-27+09:27&entry.1917517419=2024-04-22+20:07&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**